src/ssl.cc

```diff
@@ -47,11 +47,7 @@
 drizzle_return_t drizzle_set_ssl(drizzle_st *con, const char *key, const char *cert, const char *ca, const char *capath, const char *cipher)
```

Comment:
Could you rephrase the commit message, I am struggling to work out what you mean.

Reply:
Yes